firebase-firestore/src/main/java/com/google/firebase/firestore/UserDataConverter.java

```diff
@@ -117,7 +117,7 @@ public ParsedUpdateData parseUpdateData(Map<String, Object> data) {
 
     ParseAccumulator accumulator = new ParseAccumulator(UserData.Source.Update);
     ParseContext context = accumulator.rootContext();
-    ObjectValue updateData = ObjectValue.emptyObject();
+    ObjectValue.Builder updateData = ObjectValue.Builder.emptyBuilder();
```

Comment:
This is pretty verbose. How about `ObjectValue.newBuilder()`?

Reply:
Works for me. I got inspired by the rest of the Protobuf API.